src/hist/intervals.py

```diff
@@ -128,7 +128,9 @@ def ratio_uncertainty(
          numerator scaled by the denominator.
          ``"poisson-ratio"`` implements the Clopper-Pearson interval for Poisson
          distributed ``num`` and ``denom``.
```

Comment:
Should probably mention Nick's https://github.com/scikit-hep/hist/issues/277#issuecomment-892192742 to make clear that these are assumed to be independent. This will compliment the nice explanation added in "efficiency" as well.